go-controller/pkg/ovn/egressip_test.go

```diff
@@ -170,6 +170,12 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations", func() {
 		return reAssignmentCount
 	}
 
+	getEgressIPsCount := func() int {
+		egressIPs, err := fakeOvn.controller.watchFactory.GetEgressIPs()
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		return len(egressIPs)
+	}
```

Comment:
I would actually not even bother with defining this function.

I think you can simply add the following expectation, which will have the same outcome:
```golang
gomega.Eventually(fakeOvn.controller.watchFactory.GetEgressIPs).Should(gomega.Equal(1))
```
since gomega's assertion funcs (`Should` / `To` / ...) accept [multiple return arguments](https://github.com/onsi/gomega/blob/5f71ed29d5d3935fe131aa4f78bf399a7f1d71ec/gomega_dsl.go#L279). Quote from their docs:
> If multiple values are returned by the function, Eventually will pass the first value to the matcher and require that all others are zero-valued.  This allows you to pass Eventually a function that returns a value and an error - a common pattern in Go.

Reply:
hm... I will try, but what I need is to get the len() of that first value. Would what still work?

Reply:
If you need `len`, you can use the [HaveLen](https://github.com/onsi/gomega/blob/master/matchers/have_len_matcher.go#L9) matcher instead.

It likely provides a better experience when debugging.